.github/workflows/test-comment.yml

```
@@ -0,0 +1,42 @@
+name: Test Comment Trigger
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  debug:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Debug all info
+        run: |
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)